<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabMaintenanceAutoReboot</class>
 <widget class="QWidget" name="TabMaintenanceAutoReboot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ComboBox" name="comboBox_day"/>
     </item>
     <item row="3" column="1">
      <widget class="MyTimeEdit" name="timeEdit_time">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_day">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="comboBox_enable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_autoReboot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Auto Reboot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>638</width>
       <height>268</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>mytimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
